server/index.js

```diff
@@ -68,6 +92,95 @@ app.get('/api/venues/:city/:search', (req, res, next) => {
       res.send(formatData(json));
     });
 });
+app.get('/create', (req, res, next) => {
+
+  connection.query("DELETE FROM favorites WHERE venue_json = '456'", function (err) {
+    if (err) {
+      console.error(err);
+    }
```

Comment:
If there is an error, you should still send a 500 response, just like in all of our express exercises. check all of your server code to make sure that the client receives a 500 response when there is an error like this